lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -169,7 +170,7 @@
   end
 
   execute("git-remove-delivery-config-branch") do
-    command("git branch -d add-delivery-configuration")
+    command("git branch -D add-delivery-configuration")
```

Comment:
@danielsdeleo unless we do some crazy git stuff to change the name of master, having pipelines besides master will hit issues with the branch not being merged (even though it was merged into mainline or some other name for "master").